### PR TITLE
Do not use deprecated eslint rules

### PR DESCRIPTION
**Pull request by cezaraugusto**
See https://eslint.org/docs/user-guide/migrating-to-7.0.0#nodejscommonjs-rules-have-been-deprecated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
     "eslint": "^7.13.0",
     "lerna": "^3.22.1",
     "prettier": "^2.2.0"
+  },
+  "dependencies": {
+    "eslint-plugin-node": "^11.1.0"
   }
```
Comment (kaisermann) [inline package.json:21]: Remove this from the root's package.json. It should be in the `packages/eslint-config/package.json` file

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (cezaraugusto) [reply package.json]: done

**Review by kaisermann — APPROVED** [review]

**Comment by kaisermann** [comment]
niiiiiiiiiiice 🆗

File: packages/eslint-config/rules/node.js [hunk 2]
```diff
@@ -2,26 +2,28 @@ module.exports = {
...
 
     // Disallow use of new operator with the require function
-    // https://eslint.org/docs/rules/no-new-require
-    'no-new-require': 'error',
+    // https://github.com/mysticatea/eslint-plugin-node/blob/v11.1.0/docs/rules/no-new-require.md
+    'node/no-new-require': 'error',
 
     // Disallow string concatenation with __dirname and __filename
-    // https://eslint.org/docs/rules/no-path-concat
-    'no-path-concat': 'error',
```
Comment (kaisermann) [inline packages/eslint-config/rules/node.js:25]: Can you undo the changes from 'off' to 'error'?

**Review by kaisermann — CHANGES_REQUESTED** [review]

**Review by kaisermann — APPROVED** [review]
